atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionSerializableConflictException.java

```diff
@@ -16,25 +16,43 @@
 package com.palantir.atlasdb.transaction.api;
 
 import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.logsafe.Arg;
+import com.palantir.logsafe.Safe;
+import com.palantir.logsafe.SafeLoggable;
+import java.util.List;
 
-public class TransactionSerializableConflictException extends TransactionFailedRetriableException {
+public class TransactionSerializableConflictException extends TransactionFailedRetriableException
+        implements SafeLoggable {
     private static final long serialVersionUID = 1L;
 
     private final TableReference conflictingTable;
+    private final List<Arg<?>> args;
 
-    public TransactionSerializableConflictException(String message, TableReference conflictingTable) {
+    public TransactionSerializableConflictException(
+            String message, TableReference conflictingTable, List<Arg<?>> args) {
         super(message);
         this.conflictingTable = conflictingTable;
+        this.args = List.copyOf(args);
     }
 
     public static TransactionSerializableConflictException create(
-            TableReference tableRef, long timestamp, long elapsedMillis) {
+            TableReference tableRef, long timestamp, long elapsedMillis, List<Arg<?>> args) {
```

Comment:
We always pass only a list of one. I am assuming we use a list to support future cases where we want to add more args? Also, might be interesting to play with a variadic number of arguments for Arg instead of list - may make the test fixes look better

Reply:
> Also, might be interesting to play with a variadic number of arguments for Arg instead of list - may make the test fixes look better

Up to you, but this is marginal more forwards compatible, since you can only have a single varargs method.